.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:9.11
```

Comment:
why we need node on this machine?

Reply:
it is pretty light and has git, ssh and npm